[BD-05][EDUCATOR-5596] Remove tinymcev5 and use the existing one from the platform

shimulch opened the pull request:
During testing, it has been discovered that the TinyMCE v5 used in the Pluggable ORA Submission editor collides with the existing TinyMCE v4 in the platform. This PR removes TinyMCE v5 and uses the version from the platform instead.

**JIRA tickets**: https://openedx.atlassian.net/browse/EDUCATOR-5596

~~**Discussions**:~~

**Dependencies**: None

**Screenshots**:
<img width="938" alt="Screenshot 2021-02-26 at 1 09 32 PM" src="https://user-images.githubusercontent.com/1010244/109268551-6fd8a180-7835-11eb-8d7c-a18e508f9a65.png">

<img width="856" alt="Screenshot 2021-02-26 at 1 11 29 PM" src="https://user-images.githubusercontent.com/1010244/109268565-7535ec00-7835-11eb-8f9c-3f557c09045c.png">


**Sandbox URL**: TBD

**Merge deadline**: None

**Testing instructions**:

1. Pull this PR into your local devstack.
2. Install `edx-ora2`.
3. Create a ORA block in studio.
4. From Edit -> Settings -> Response Editor, select WYSIWYG Editor.
5. Check that TinyMCE Response Editor loads in Studio.
6. Check that ORA Prompt editor works.
7. Publish the block.
8. Check TinyMCE loads in LMS.

**Author notes and concerns**:

TBD

**Reviewers**
- [ ] @giovannicimolin 
- [ ] edX reviewer[s] TBD

edx-webhook commented:
Thanks for the pull request, @shimulch! I've created [BLENDED-778](https://openedx.atlassian.net/browse/BLENDED-778) to keep track of it in Jira.   More details are on the [BD-05](https://openedx.atlassian.net/wiki/spaces/COMM/pages/1529970851/BD-05+ORA+v3+Usability+Moderation) project page.


This is currently a draft pull request. When it is ready for our review and all tests are green, click "Ready for Review", or remove "WIP" from the title, as appropriate.

<!-- comment:end_of_wip -->


<!-- comment:welcome-blended -->
<!-- jenkins ok to test -->
<!-- data: eyJkcmFmdCI6IHRydWV9 -->

giovannicimolin commented on openassessment/xblock/static/js/src/lms/editors/oa_editor_tinymce.js:
```diff
@@ -50,18 +62,13 @@ tinymce.baseURL = '/xblock/resource/openassessment/static/vendors/tinymce/';
 
         const ctrl = this;
-
+        // this.elements.tinymce()
```
@shimulch Please remove this comment

shimulch replied:
Removed

giovannicimolin commented on openassessment/xblock/static/js/src/lms/editors/oa_editor_tinymce.js:
```diff
@@ -1,15 +1,23 @@
+  const tinymceCssFile = '/static/js/vendor/tinymce/js/tinymce/skins/studio-tmce4/skin.min.css';
+
+  if (typeof window.tinymce === 'undefined') {
+    // as tinymce is not available, we need to load them
```
```suggestion
    // If tinymce is not available, we need to load it
```

shimulch replied:
Updated.

giovannicimolin reviewed (changes requested):
Minor nits. 

Can you also remove `tinymce` from the `package.json` file and add a version bump?

shimulch commented:
@giovannicimolin, I've added content styling CSS for tinyMCE. Hopefully, it will look a bit nicer now.
Also, removed tinymce from package.json & added version bumps.

giovannicimolin reviewed (approved):
:+1:

- [x] I tested this:
1. Tested that the editor works on LMS and studio.
2. Checked that text is nicely displayed in the readonly version of the editor.
- [x] I read through the code
- [x] I checked for accessibility issues: fixes issues caused by previous implementation.
- [ ] ~~Includes documentation~~ NA

shimulch commented:
@giovannicimolin, I've tested your commit that fixes the whitespace issue. LGTM 👍🏽 Thanks for figuring out a way to achieve that.

jnlapierre commented:
Please resolve merge conflicts and we will finalize the review.

jnlapierre reviewed (approved):
LGTM
